.changeset/sixty-balloons-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-node-test-utils': patch
+---
+
+Add an initiator credentials getter to the default mock context.
```

Comment:
Is this breaking @benjdlambert ?

Reply:
Nope :tada: that's the nice thing about it :pray: